src/plugins/search/view.js

```diff
@@ -2,11 +2,12 @@ class SearchView {
   /**
    * @param {object} params
    * @param {object} params.br The BookReader instance
+   * @param {function} params.cancelSearch callback when a user wants to cancel search
    *
    * @event BookReader:SearchResultsCleared - when the search results nav gets cleared
    * @event BookReader:ToggleSearchMenu - when search results menu should toggle
    */
-  constructor({ br }) {
+  constructor({ br, cancelSearch = () => {} }) {
```

Comment:
_Minor_ I might call this `searchCancelledCallback` since that seems like when it'll be executed